components/Header/src/Header.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { HTMLAttributes } from 'react';
+import clsx from 'clsx';
+import './index.scss';
+
+export type HeaderProps = HTMLAttributes<HTMLDivElement>;
+
+export const Header: React.FC<HeaderProps> = (props: HeaderProps) => {
```

Comment:
I prefer destructuring the props instead of using `props.className`, do you prefer it the other way around?